src/vdom/VNode.ts

```
@@ -56,6 +58,14 @@ export class VNode implements Comparable<VNode> {
         if(!this.attrs.some(attr => attr.attrName == kloudAppId)) {
             this.attrs.push(new Attribute(kloudAppId, this.id));
         }
+
+        this.resolvei18n().catch(e => {
```

Don't route to error here, rather just let it fail. If another resolver is able to complete the request it should be fine

---

Done